operator/src/main/resources/scripts/modelInImage.sh

```diff
@@ -260,6 +262,57 @@ function buildWDTParams_MD5() {
     OPSS_FLAGS=""
   fi
 
+  overrideWDTTimeoutValues
+
+  trace "Exiting setupInventoryList"
+}
+
+function overrideWDTTimeoutValues() {
+  trace "Entering overrideWDTTimeoutValues"
+
+  # WDT defaults
+  #
+  #  connect.timeout=120000
+  #  activate.timeout=180000
+  #  deploy.timeout=180000
+  #  redeploy.timeout=180000
+  #  undeploy.timeout=180000
+  #  start.application.timeout=180000
+  #  stop.application.timeout=180000
+  #  set.server.groups.timeout=30000
+
+  if [ ! -z ${wdt_connect_timeout} ] ; then
+    sed -i "s/\(connect\.timeout=\).*\$/\1${wdt_connect_timeout}/" ${WDT_ROOT}/lib/tool.properties
```

Comment:
minor code refactor:  to reduce code repetition, could you create a helper function that takes the env var name and property name?  and then call it for each property?

Reply:
done